pkg/network/node/pod.go

```diff
@@ -42,6 +43,7 @@ import (
 
 const (
 	podInterfaceName = knetwork.DefaultInterfaceName
+	cniBinPath       = kcni.DefaultCNIDir
```

Comment:
As a follow up, we could add a validation to check kubeletArguments["cni-bin-path"] is not set when openshift network plugin is used.